code/modules/emotes/definitions/audible.dm

```diff
@@ -270,3 +270,15 @@
 	emote_message_1p_target = "You prbt at TARGET."
 	emote_message_3p_target = "prbts at TARGET."
 	emote_sound = 'sound/voice/prbt.ogg'
+
```

Comment:
Needs to be moved to modular or requires //CHOMPAdd start and //CHOMPAdd end comments